docs/UserGuide.md

```diff
@@ -198,7 +198,7 @@ Format: `edit -c INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]… [rt/T
 * Tags that are not mentioned will *NOT* be affected.
 * `t/TAG` adds `TAG` to the user.
 * `rt/TAG` removes `TAG` from the user.
-* Tag removal is done before new tags are added.
+* Tags are removed before new tags are added.
```

Comment:
Please leave the original. If you want to change this, it would be more precise to say 'Tag removal is done before tag addition'.

Reply:
Changed as recommended.